pipelines/skylab/optimus/Optimus.wdl

```diff
@@ -87,7 +87,8 @@ workflow Optimus {
       white_list = whitelist,
       tar_star_reference = tar_star_reference,
       chemistry = chemistry,
-      counting_mode = counting_mode
+      counting_mode = counting_mode,
+      input_id = input_id
```

Comment:
Should we delete the old input variable that isn't used:
 `String output_bam_basename = input_id`

Reply:
```suggestion
input_id = output_bam_basename
```